src/test/regress/expected/multi_modifying_xacts.out

```diff
@@ -357,7 +357,7 @@ ORDER BY nodeport;
 (2 rows)
 
 -- register after insert trigger
-SELECT * FROM run_command_on_placements('researchers', 'CREATE CONSTRAINT TRIGGER reject_large_researcher_id AFTER INSERT ON %s DEFERRABLE INITIALLY DEFERRED FOR EACH ROW EXECUTE FUNCTION  reject_large_id()')
+SELECT * FROM run_command_on_placements('researchers', 'CREATE CONSTRAINT TRIGGER reject_large_researcher_id AFTER INSERT ON %s DEFERRABLE INITIALLY DEFERRED FOR EACH ROW EXECUTE PROCEDURE  reject_large_id()')
```

Comment:
There was a normalization rule https://github.com/citusdata/citus/pull/4799/files/d788ab6ba9d8fb20f08f6842c843a21678f29516#diff-642588ef740a887897c3276d0b0c01f1fa12577d617b806e288f5669f5863190L50. It is because of that

(in case the link is broken):
`s/FOR EACH (ROW|STATEMENT)(.*)EXECUTE PROCEDURE/FOR EACH \1\2EXECUTE FUNCTION/g`